tests/System/Controller/SecuredViewHandlerTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+declare(strict_types=1);
+namespace PhpList\RestBundle\Tests\System\Controller;
+
+use GuzzleHttp\Client;
+use PhpList\Core\TestingSupport\Traits\SymfonyServerTrait;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Test for security headers
+ * @author Xheni Myrtaj <xheni@phplist.com>
+ *
+ *
+ */
+class SecuredViewHandlerTest extends TestCase
+{
+
```

Comment:
Please let's drop the blank line. (In general, there should be no blank line below a `{` or above a `}`.)